### ISSUE TITLE
All processes of a MPI task land on the same node with orte launcher

### ISSUE THREAD

**Issue by vivek-bala** [radical-cybertools/radical.pilot#1729]
Launch cmd in shell script:
```
/lustre/atlas2/csc230/world-shared/openmpi//installed/2018_07_26_539f71d/bin/orterun  --hnp "2988965888.0;tcp://10.128.36.167,160.91.205.195:56402" -np 8 --bind-to none -host 16694,16695,16616,16617,16618,16619,16692,16693 -x "LD_LIBRARY_PATH" -x "PATH" -x "PYTHONPATH"  python "mpi_stress.py" "16" "60"  1>/lustre/atlas2/csc190/scratch/vivekb/radical.pilot.sandbox/re.session.titan-ext4.vivekb.017802.0007/pilot.0000/unit.000005/STDOUT 2>/lustre/atlas2/csc190/scratch/vivekb/radical.pilot.sandbox/re.session.titan-ext4.vivekb.017802.0007/pilot.0000/unit.000005/STDERR
```

Output from task:
```
> cat STDOUT 
[ORTE] Task: 0 is launched! (Job ID: [45608,7])
Hello! I'm rank 0 from 8 running on nid16694
Hello! I'm rank 2 from 8 running on nid16694
Hello! I'm rank 6 from 8 running on nid16694
Hello! I'm rank 7 from 8 running on nid16694
Hello! I'm rank 1 from 8 running on nid16694
Hello! I'm rank 5 from 8 running on nid16694
Hello! I'm rank 4 from 8 running on nid16694
Hello! I'm rank 3 from 8 running on nid16694
[ORTE] Task: 0 returned: 0 (Job ID: [45608,7])
```

Stack:
```
python               : 2.7.9
pythonpath           : /sw/xk6/xalt/0.7.5/site:/sw/xk6/xalt/0.7.5/libexec
virtualenv           : /autofs/nccs-svm1_home1/vivekb/ves/ve-bms

radical.entk         : 0.7.6
radical.pilot        : 0.50.8-v0.50.8-13-gdc2720af@feature-shellfs_spawner
radical.utils        : 0.50.2
saga                 : 0.50.0
```

Note that the above script worked as expected with aprun:
```
aprun -n 8 -d 16 python "mpi_stress.py" "16" "60" 
```

Agent scheduler: Continuous
Script run by task: https://gist.github.com/vivek-bala/f07069c80217a467dd2c0305f89d8a33
Pilot sandbox location: /lustre/atlas/world-shared/csc190/re.session.titan-ext4.vivekb.017802.0007

Let me know if additional data is required to debug

**Comment by vivek-bala** [comment]
I didn't realize this was not branched off the shellfs spawner branch. I couldn't understand why this branch was failing for large number of tasks. Could these two (1729 and shelffs branches) be merged? This branch by itself is working for small number of tasks.

**Comment by vivek-bala** [comment]
FYI I merged the two branches locally and used it on Titan. The merged branch behaves as expected.

**Comment by lee212** [comment]
@vivek-bala, I believe this issue has been resolved in the branch `fix/issue_1729` and the shellfs spawner (#1714) is also merged earlier?

**Comment by mturilli** [comment]
Porte is now obsolete.
